src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddress.java

```diff
@@ -26,8 +29,8 @@
 
     private final String identifier;
     private final HealthcareRegistrationIdentificationCode code;
-    private final String performingOrganisationName;
     private final String performerName;
+    private final String performingOrganisationName;
```

Comment:
Since this class is already `PerformerNameAndAddress`  perhaps we don't need the `performer` and `performing` prefixes here.

```suggestion
    private final String name; // or individualName?
    private final String organisationName;
```